src/lib/metrics.ts

```diff
@@ -351,6 +351,15 @@ export default class MetricsMonitor {
                 rateLimits
                     .labels({ endpoint: '/auth/simple', method: 'POST' })
                     .set(config.rateLimiting.simpleLoginMaxPerMinute);
+                rateLimits
+                    .labels({
+                        endpoint: '/api/incoming-webhook/:name',
+                        method: 'POST',
+                    })
+                    .set(
+                        config.rateLimiting.callIncomingWebhookMaxPerSecond *
+                            60,
```

Comment:
Current rate limit unit in metrics is "per minute", so we `* 60` the configured RPS value to convert to RPM.